docs/phenopacket.rst

```diff
@@ -25,35 +25,35 @@ describe the phenotypic abnormalities directly associated with an extirpated or
     * - id
       - string
       - required
-      - arbitrary identifier
+      - Arbitrary identifier
     * - subject
       - :ref:`rstindividual`
       - recommended
       - The proband
     * - phenotypic_features
-      - List of :ref:`phenotypicfeature`
+      - :ref:`phenotypicfeature` (list)
       - recommended
       - Phenotypic features observed in the proband
     * - biosamples
       - :ref:`rstbiosample`
       - optional
-      - samples (e.g., biopsies), if any
+      - Samples (e.g., biopsies), if any
     * - genes
       - :ref:`rstgene`
       - optional
       - Gene deemed to be relevant to the case (application specific)
     * - variants
-      - List of :ref:`rstvariant`
+      - :ref:`rstvariant` (list)
       - optional
       - Variants identified in the proband
     * - diseases
-      - List of :ref:`rstdisease`
+      - :ref:`rstdisease` (list)
       - optional
       - Disease(s) diagnosed in the proband
     * - hts_files
-      - List of :ref:`rstfile`
+      - :ref:`rstfile` (list)
       - optional
-      - VCF or other high-throughput sequencing files
+      - High-throughput sequencing files (ie. VCF)
```

Comment:
Replace 'i.e.' with 'e.g.' as in 'for example'. To quote https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference

> i.e. is an abbreviation for the phrase id est, which means "that is." I.e. is used to restate something said previously in order to clarify its meaning. E.g., on the other hand, is short for exempli gratia, which means "for example." E.g. is used before an item or list of items that serve as examples for the previous statement.

Reply:
Done